xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -1376,6 +1376,11 @@ core.widgets.suggest.showResults=Go to search page\u2026
 core.widgets.suggest.valuePrefix=Value:
 core.widgets.suggest.transportError=Failed to retrieve suggestions:
 core.widgets.suggest.hide=hide suggestions
+core.widgets.suggest.attachmentPicker.title=Pick attachments.
+core.widgets.suggest.classPicker.title=Pick classes.
+core.widgets.suggest.pagePicker.title=Pick pages.
+core.widgets.suggest.userPicker.title=Pick users.
+core.widgets.suggest.xpropertyPicker.title=Pick property values.
```

Comment:
I'm wondering about the single vs. multiple select use case. As far as I understand, all of them support single or multiple selection. I think the label should reflect this.

Reply:
:+1: 
Fixed in https://github.com/xwiki/xwiki-platform/pull/2221/commits/dbfb5ddb85158d89a0730f12c2f1c57a15459357